benchmark/main.cpp

```diff
@@ -747,6 +747,7 @@ MBED_NOINLINE static int benchmark_ctr_drbg()
 {
     int ret = 0;
     const char *nopr_title = "CTR_DRBG (NOPR)";
+    const char *pr_title = "CTR_DRBG (PR)";
```

Comment:
why not use the existing `title` buffer, and just use the snprintf to write to it the relevant title?

Reply:
I am not sure what is the point of using title and snprintf(). There is nothing to format here, these two variables are just a short hand to not have to write the string twice in the function body.